### DISCUSSION TITLE
Hello i'm a developper but i have problems like i don't now whay am good or not i'm always negative about my compétences and i'm searching for job any tips for me ?

### DISCUSSION THREAD

**Discussion by SNcodeur2001** [community/community#143206] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i'm Open

**Comment by THE-DEEPDAS** [comment, marked as answer]
Hello bro,It’s very normal to feel uncertain, especially in tech, where there’s always so much to learn. Here are some ways to boost your confidence and make your job search feel more manageable:

Focus on Your Achievements: Make a list of projects you’ve completed, challenges you’ve overcome, and skills you’ve learned. When you see these accomplishments in front of you, it’s easier to recognize your value.

Practice Coding Regularly: Try daily coding challenges or work on side projects. This will not only improve your skills but also give you more to show in your portfolio, which is often reassuring when job hunting.

Break Down Your Skill Set: Make a list of your strengths and areas you want to improve. This approach is constructive and gives you specific goals to work toward, like improving in a particular language or learning new frameworks.

Network and Find Supportive Communities: Join developer forums, LinkedIn groups, or even local meetups. Talking to others often helps put your experience in perspective and provides encouragement.

Apply with Positivity: Start your job search with the mindset that not every application has to be perfect. Focus on jobs where you meet at least 70% of the requirements, and don’t be discouraged by rejections. Each interview is valuable experience.

**Reply by SNcodeur2001** [reply]
Thank you very I'm gonna do this

**Reply by if-i-shajan** [reply]
Wish you all the best.

**Reply by Presecan88** [reply]
Thanks for the advice!

**Comment by prathamlashkari** [comment]
Hey, I totally agree with you! I’m from the 2025 batch and struggled to grab an internship during my three years. But I kept practicing and, with some help, finally landed one. During that time, I made lots of connections and friends at the company, and I even secured a pre-placement offer (PPO). So, just focus on building connections and don’t hesitate to ask for help. If you have the right skills, people will be willing to assist you, which can really help in landing a job. I got mine by reaching out too.
